lib/cocoapods-core/specification.rb

```diff
@@ -203,6 +207,16 @@ def subspec?
 
     # @!group Dependencies & Subspecs
 
+    # @return [Bool] if the specification is a test specification
+    def test_specification?
+      @test_specification
+    end
+
+    # @return [Symbol] the test type supported if this is a test specification
+    def test_type
```

Comment:
Should this be declared here? I see there are definitions in `root_attribute_accessors.rb` but this is not a root attribute. Where do the "getters" go?